### PR TITLE
Removing Jurisdiction Requirements

### PR DESCRIPTION
**Pull request by freddy-m**
Closes: https://github.com/privacytools/privacytools.io/issues/1437

### CODE REVIEW

**Comment by dngray** [comment]
We probably want to write a blog post about this before we merge and also update the VPN page's criteria, it was discussed quite thoroughly https://github.com/privacytools/privacytools.io/issues/1437

File: docs/email.md [hunk 1]
```diff
@@ -295,19 +295,6 @@ Tutanota is working on a [desktop client](https://tutanota.com/blog/posts/deskto
 
 **Please note we are not affiliated with any of the providers we recommend.** This allows us to provide completely objective recommendations. We have developed a clear set of requirements for any Email provider wishing to be recommended, including implementing industry best practices, modern technology and more. We suggest you familiarize yourself with this list before choosing an Email provider, and conduct your own research to ensure the Email provider you choose is the right choice for you.
```
Comment (EdwardLangdon) [inline docs/email.md:297]: Considering adding Why removed the requirements and reason not to Consider about `Eyes` Countries.

Comment (dngray) [reply docs/email.md:297]: I intend to write a blog post about it actually.

Comment (EdwardLangdon) [reply docs/email.md:297]: Okay, Then Hyperlink your post after you have completed writing.

**Review by jonaharagon — CHANGES_REQUESTED** [review]
Needs to be removed on VPN.md as well.

**Comment by dngray** [comment]
> Needs to be removed on VPN.md as well.

Yep, we're looking at https://github.com/privacyguides/privacyguides.org/pull/960#issuecomment-1097927203 first as i want to get some of the threat modeling stuff  done first, that way when we look at vpn/email we can refer to threat modeling page instead.

**Comment by dngray** [comment]
> Needs to be removed on VPN.md as well.

Okay, done, now we've got the threat modeling page, this makes a bit more sense.

**Review by dngray — APPROVED** [review]

**Pull request merged**